src/pages/home/report/ReportActionCompose.js

```diff
@@ -372,7 +373,8 @@ class ReportActionCompose extends React.Component {
      * @param {String} newComment
      */
     updateComment(newComment) {
-        this.textInput.setNativeProps({text: newComment});
+        const textWithEmojis = EmojiUtils.replaceEmojis(newComment);
+        this.textInput.setNativeProps({text: textWithEmojis});
```

Comment:
Can we just simplify the first few lines
```js
    /**
     * Update the value of the comment in Onyx
     *
     * @param {String} comment
     */
    updateComment(comment) {
        const newComment = EmojiUtils.replaceEmojis(comment);
```

Reply:
Removed all empty line comments from the entire file.

Reply:
I think you might've misinterpretted @Santhosh-Sellavel's suggestion here. I think he was mostly commenting on variable naming, like so:

```diff
+ @param {String} comment
+ updateComment(comment) {
+     const newComment = EmojiUtils.replaceEmojis(comment);
+     this.textInput.setNativeProps({text: newComment});
- @param {String} newComment
- updateComment(newComment) {
-     const textWithEmojis = EmojiUtils.replaceEmojis(newComment);
-     this.textInput.setNativeProps({text: textWithEmojis});
```

Reply:
I suggest leaving it as it is because there is a variable inside the class component named `comment` and we are using it inside the method body(we need to distinguish between them), but if you see we should rename `newComment` to `comment` no problem at all.
https://github.com/Expensify/App/blob/69b735716ae444dd5b8a1d227b5ca649e70553c5/src/pages/home/report/ReportActionCompose.js#L381

Reply:
It shouldn't be a problem.

Reply:
@Santhosh-Sellavel I got your point, you are right you are looking for minimal changes, sorry for the misunderstanding.